EFCore.SqlServer.HierarchyId.Test/EFCore.SqlServer.HierarchyId.Test.csproj

```diff
@@ -1,20 +1,20 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net7.0</TargetFramework>
```

Comment:
EF Core 7 targets net 6

Reply:
For Complete Test of this project we should change target framework of Test Project to `net7.0` and use visual studio preview  to build